testing/ef-tests/src/models.rs

```diff
@@ -129,6 +129,10 @@ pub struct Block {
     pub block_header: Option<Header>,
     /// RLP encoded block bytes
     pub rlp: Bytes,
+    /// If the execution of the block should fail
+    /// `expect_exception` is `Some`.
+    /// Its contents detail the reason for the failure.
+    pub expect_exception: Option<String>,
```

Comment:
invalid_blocks test case has this field in its JSON -- however, the tests are more nuanced than what we are testing here; some blocks do execute successfully -- so its possible to assert that the first 3 blocks successfully executed and the rest were unsuccessful